tracer/build/_build/docker/smoke.nuget.dockerfile

```
@@ -12,7 +12,7 @@ COPY ./test/test-applications/regression/AspNetCoreSmokeTest/ .
 RUN dotnet restore "AspNetCoreSmokeTest.csproj"
 
 # install the package
-RUN dotnet add package "Datadog.Monitoring.Distribution" --prerelease --source /src/artifacts
+RUN dotnet add package "Datadog.Trace.Bundle" --prerelease --source /src/artifacts
```

No need for the prerelease anymore
```suggestion
RUN dotnet add package "Datadog.Trace.Bundle" --source /src/artifacts
```